registry.json

```diff
@@ -3043,6 +3043,36 @@
       "documentation": "https://docs.thorchain.org"
     }
   },
+  {
+    "id": "mayachain",
+    "name": "MAYAChain",
```

Comment:
Why did you decide to use upper-case? They use `Maya` naming everywhere

Reply:
Maya Protocol encompasses two chains, MAYAChain and AZTECChain. The upper-case is just our naming convention haha.

Reply:
Yes, as @GiMa-Maya said that's our naming convention but if you prefer we can change it to lowercase